admiral/pkg/controller/secret/secretcontroller.go

```diff
@@ -139,6 +145,13 @@ func NewController(
 				queue.Add(key)
 			}
 		},
+		UpdateFunc: func(oldObj interface{}, newObj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(newObj)
+			log.Infof("Processing update: %s", key)
```

Comment:
Change the log line to "Processing secret update for cluster: %s" for better readability.